### ISSUE TITLE
apply css class for specific label

### ISSUE THREAD

**Issue by tentamen** [bilus/reforms#8]
the form has some mandatory input fields. I want to add `control-madatory` class to these label only.

how can i do that?

**Comment by bilus** [comment]
While there is no way to add class to a label, how about reshaping your css a bit:

```css
input.control-mandatory label {
  ...
}

**Comment by bilus** [comment]
This is kinda hack and pretty verbose but you should be able to use `with-options`:

```clojure
(form
   ...
   (with-options {:form
                  {:label-column-class
                   "col-sm-3 control-mandatory"}}
     (text ...)))
```

If it works, you can create a wrapper macro that does the trick (untested):

```clojure
(defmacro mandatory
    [control]
    `(with-options {:form
                    {:label-column-class
                     "col-sm-3 control-mandatory"}}
       ~control))
```

and use it like this:

```clojure
(form
   ...
   (mandatory (text ...)))
```

Please let me know if this is a satisfactory solution. Also, if you have any suggestions as to how you think would be the ideal API for this, please do share it.

**Comment by tentamen** [comment]
Thanks for your solution. It's work!

having a `:label-class` option for each control would be nice, i think. Does it just adding css class to `:label-column-class` form options (or overwrite), in your opinion?

**Comment by bilus** [comment]
Good question. I thought `:label-class` would just add to whatever is in `:label-column-class` so maybe calling it `:label-addon-class` would be more intuitive?

**Comment by tentamen** [comment]
Agreed. I'd prefer having `:label-addon-class` in args of control function, than do it in `with-options`.

**Comment by bilus** [comment]
Yes, I can see that would be nice but difficult to pull off and keep the API clean. I'll think about it but for the time being, I think I'll settle for adding `:label-addon-class` with `with-options` (or `set-options!`). But I'll think about it before making any changes, I promise. :)
